src/index.js

```diff
@@ -118,36 +142,44 @@ function processDdoc(ddoc) {
       .then(() => lockfile.release());
   } else {
     info('No apps have changed.');
-    return Promise.resolve();
+
+    if (firstRun) {
+      return startApps();
+    } else {
+      return Promise.resolve();
+    }
   }
 }
 
 
-function bootstrapDdoc() {
-  if(!shouldBootstrapDdoc) {
-    return info('No bootstrap requested.');
-  } else {
-    info('Bootstrap requested.');
-    trace(`Fetching new ddoc from ${STAGING_URL}…`);
-    return new PouchDB(STAGING_URL)
-      .get('master', { attachments:true })
-      .then(newDdoc => {
-        trace('New ddoc fetched.');
-        newDdoc._id = '_design/medic';
-        delete newDdoc._rev;
-        trace('Fetching old ddoc from local db…');
-        return db
-          .get(DDOC)
-          .then(oldDdoc => newDdoc._rev = oldDdoc._rev)
-          .catch(err => {
-            if(err.status === 404) trace('No old ddoc found locally.');
-            else throw err;
-          })
-          .then(() => trace('Uploading new ddoc to local db…'))
-          .then(() => db.put(newDdoc))
-          .then(() => trace('Bootstrap complete.'));
-      });
-  }
+function bootstrap() {
+  info(`Bootstrap requested. Bootstrapping to ${bootstrapDdoc}`);
+  trace(`Fetching new ddoc from ${STAGING_URL}…`);
+  return new PouchDB(STAGING_URL)
+    .get(`medic:medic:${bootstrapDdoc}`, { attachments:true })
+    .then(newDdoc => {
+      trace('New ddoc fetched.');
+      newDdoc._id = STAGED_DDOC;
+      newDdoc.deploy_info = {
+        timestamp: new Date().toString(),
+        user: 'horticulturalist (bootstrap)',
+        version: bootstrapDdoc,
+      };
+      delete newDdoc._rev;
+      trace('Fetching old staged ddoc from local db…');
+      return db
+        .get(STAGED_DDOC)
+        .then(oldDdoc => newDdoc._rev = oldDdoc._rev)
+        .catch(err => {
+          if (err.status === 404) trace('No old staged ddoc found locally.');
+          else throw err;
+        })
+        .then(() => trace('Uploading new ddoc to local db…'))
+        .then(() => db.put(newDdoc))
+        .then(() => db.get(STAGED_DDOC, {attachments: true}))
```

Comment:
Yeah good point, I don't know why I'm doing this. I may be because I'm pulling the attachments, but I could do that at `:171` and I think it should work. I'll investigate, removing it if I don't need it and documenting it if I do.